app/controllers/candidate_interface/course_choices_controller.rb

```diff
@@ -6,9 +6,9 @@ def index
       @application_form = current_application
       @course_choices = current_candidate.current_application.application_choices
       @page_title = if @course_choices.count < 1
-                      t('page_titles.choosing_courses')
+                      I18n.t!('page_titles.choosing_courses')
```

Comment:
Bit annoying that getting `I18n` to raise isn't globally configurable outside views.